wasm.md

```diff
@@ -1017,23 +1005,43 @@ The only allowed `TableElemType` is "funcref", so we ignore this term in the red
 
 ```k
     syntax Defn      ::= TableDefn
-    syntax TableDefn ::= "(" "table"     OptionalId Limits TableElemType ")"
-                       | "(" "table"     OptionalId        TableElemType "(" "elem" ElemSegment ")" ")"
+    syntax TableType ::= Limits TableElemType
+    syntax TableSpec ::= TableType
+                       | TableElemType "(" "elem" ElemSegment ")"
+                       | InlineImport TableType
+    syntax TableDefn ::= "(" "table"     OptionalId InlineExports TableSpec ")"
+                       | "(" "table"     OptionalId ")"
                        |     "table" "{" OptionalId Int MaxBound "}"
  // ----------------------------------------------------------------
-    rule <k> ( table ID:OptionalId MIN:Int         funcref ) => table { ID MIN .MaxBound } ... </k>
-      requires MIN <=Int #maxTableSize()
-    rule <k> ( table ID:OptionalId MIN:Int MAX:Int funcref ) => table { ID MIN MAX       } ... </k>
-      requires MIN <=Int #maxTableSize()
-       andBool MAX <=Int #maxTableSize()
-    rule <k> ( table funcref ( elem ES ) ) => ( table #freshId(NEXTID) funcref (elem ES) ) ... </k>
+    rule <k> ( table                  EXPO:InlineExports SPEC:TableSpec )
+          => ( table #freshId(NEXTID) EXPO               SPEC            )
+          ...
+         </k>
+         <nextFreshId> NEXTID => NEXTID +Int 1 </nextFreshId>
+      requires EXPO =/=K .InlineExports
+
+    rule <k> ( table ID:Identifier ( export ENAME ) EXPO:InlineExports SPEC:TableSpec )
+          => ( export ENAME ( table ID ) )
+          ~> ( table ID EXPO SPEC )
+          ...
+         </k>
+
+    rule <k> ( table .InlineExports funcref ( elem ES ) ) => ( table #freshId(NEXTID) .InlineExports funcref (elem ES) ) ... </k>
```

Comment:
No, it is dealing with the case when we need to desugar the inline `elem` definition, and we don't already have an identifier. So we need a fresh identifier, because the `elem` needs to know which table to refer to (technically, it's not necessary, but the spec puts it this way).